components/common/src/error.rs

```diff
@@ -35,6 +35,7 @@ pub enum Error {
     ArtifactIdentMismatch((String, String, String)),
     /// Occurs when there is no valid toml of json in the environment variable
     BadEnvConfig(String),
+    BadSymbolStyle(String),
```

Comment:
Should this be `BadGlyphStyle` now for consistency?

Reply:
ahh! good catch.